sdk/servicebus/Azure.Messaging.ServiceBus/README.md

```diff
@@ -53,21 +53,33 @@ To see how to authenticate using Azure.Identity, view this [example](#authentica
 
 ## Key concepts
 
+Once you've initialized a `ServiceBusClient`, you can interact with the primary resource types within a Service Bus Namespace, of which multiple can exist and on which actual message transmission takes place, the namespace often serving as an application container:
+
+* [Queue](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview#queues): Allows for Sending and Receiving of messages, ordered first-in-first-out. Often used for point-to-point communication.
```

Comment:
`ordered first-in-first-out` - Queues doesn't guaranteed the FIFO order(non-session case). Not sure if we should add this.

Reply:
Good point.. @KieranBrantnerMagee thoughts?

Reply:
I'm thinking of removing the ordered first-in-first-out bit.